app.py

```diff
@@ -1,4 +1,5 @@
 from flask import Flask, render_template, request, redirect, url_for
+from db_functions import get_db_connection, hash_password
```

Comment:
unnecessary for this line since get_db_connection is def in local and so hash_password,